modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -43,6 +43,10 @@
  * such as via the try-with-resources statement.
  */
 public final class ServiceInvoker implements AutoCloseable {
+  /**
+   * The User Agent header to attach to all outgoing HTTP requests.
+   */
+  public static final String USER_AGENT = "OG-JAVA-SDK";
```

Comment:
AWS uses the form `aws-sdk-go/1.1.0 (go1.5.2; darwin; amd64)`, presumably this is for the Go SDK.

How about `opengamma-sdk-java/version (other info)`